vulns.go

```diff
@@ -146,20 +150,23 @@ func getSnykOpenSourceIssueWithoutTickets(flags flags, projectID string, maturit
 
 					ProjectIssuePathData, err := makeSnykAPIRequest("GET", flags.mandatoryFlags.endpointAPI+"/v1/org/"+flags.mandatoryFlags.orgID+"/project/"+projectID+"/issue/"+issueId+"/paths", flags.mandatoryFlags.apiToken, nil, customDebug)
 					if err != nil {
-						log.Printf("*** ERROR *** Could not get aggregated data from %s org %s project %s issue %s", flags.mandatoryFlags.endpointAPI, flags.mandatoryFlags.orgID, projectID, issueId)
-						log.Fatalln(err)
+						log.Printf("*** ERROR *** Could not get paths data from %s org %s project %s issue %s", flags.mandatoryFlags.endpointAPI, flags.mandatoryFlags.orgID, projectID, issueId)
+						issueSkipped += "\nissue ID: " + issueId + " from project ID:" + projectID
+						continue
 					}
 					ProjectIssuePathDataJson, er := jsn.NewJson(ProjectIssuePathData)
 					if er != nil {
 						log.Printf("*** ERROR *** Json creation failed\n")
-						log.Fatalln(er)
+						issueSkipped += "\nissue ID: " + issueId + " from project ID:" + projectID
```

Comment:
not convinced this one is right. if you have an error above, ProjectIssuePathData is unlikely to be right and with the continue, you'll skip that anyways. This error er will be more about failing to convert into json, nothing to do wiht snyk responding wiht an error or not, so I would keep the log fatal thing instead.